index.js

```diff
@@ -1 +1,3 @@
-const eslint = require('eslint')
+const eslint2 = require('eslint')
```

Comment:
no-unused-vars: 'eslint2' is assigned a value but never used  <!-- Automatically posted by Lintly -->